activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/services/audit/AuditProducerIT.java

```diff
@@ -482,7 +486,7 @@ void shouldProduceEventsDuringSimpleProcessExecutionWithActor() {
             .filteredOn(it -> ActorConstants.ACTOR_TYPE.equals(it.getType()))
             .isNotEmpty()
             .extracting(IdentityLink::getUserId, IdentityLink::getDetails)
-            .containsOnly(tuple("hruser", expectedActor.getBytes()));
+            .containsOnly(tuple(id, expectedActor.getBytes()));
```

Comment:
This change affects the semantic meaning of the audit event that switches the user name with user guid. It will cause a breaking change downstream. It is better to use a new identity link record with another type to enrich the process instance context with new information from the provided principal oauth access token and use it to map to audit specific event schema dowstream.